manifest.json

```
@@ -1,55 +1,56 @@
 {
     "name": "Galette",
     "id": "galette",
+    "packaging_format": 1,
+    "url": "https://www.galette.eu",
+    "license": "GPL-3.0-or-later",
+    "version": "0.9.2.1~ynh1",
+    "requirements": {
+        "yunohost": ">= 3.6"
+    },
+    "services": [
+        "nginx",
+        "php5-fpm",
```

```suggestion
        "php-fpm",
```

(or php7-fpm or php7.1-fpm, idk, but certainly not php5-fpm anymore please ;P )

---

true, I wanted to change it for php7.1-fpm but it fails automated testing because of this: https://github.com/YunoHost/package_linter/blob/master/package_linter.py#L386

I'm not aware of any usage for this services group in the manifest, could it break something?